src/NodeTypeResolver/PHPStan/TypeHasher.php

```diff
@@ -94,7 +94,7 @@ private function createUnionTypeHash(UnionType $unionType): string
         $normalizedUnionType = clone $unionType;
 
         // change alias to non-alias
-        $normalizedUnionType = TypeTraverser::map(
+        TypeTraverser::map(
```

Comment:
this `TypeTraverser::map` from line 96 to 106 seems no longer needed, then we can see if that broke something, so we can revert early :)

Reply:
You can just remove exactly line 96 to 106, the clone seems still needed.